kustomize/commands/create/create_test.go

```diff
@@ -134,6 +135,22 @@ func TestCreateWithNameSuffix(t *testing.T) {
 	}
 }
 
+func TestCreateHasDefaultMetadata(t *testing.T) {
+	fSys := filesys.MakeEmptyDirInMemory()
+	fSys.WriteFile("foo.yaml", []byte(""))
+	fSys.WriteFile("bar.yaml", []byte(""))
+	opts := createFlags{resources: "foo.yaml,bar.yaml"}
+	err := runCreate(opts, fSys, factory)
+	if err != nil {
+		t.Errorf("unexpected cmd error: %v", err)
+	}
+	m := readKustomizationFS(t, fSys)
+	expected := konfig.DefaultKustomizationMetadata()
+	if !reflect.DeepEqual(m.MetaData, expected) {
+		t.Fatalf("expected %+v but got %+v", expected, m.Resources)
+	}
```

Comment:
small nit: this could be `require.Truef()` instead. Both `require.NoErrorf()` and `require.Truef()` allow an extra argument for an error message.

Reply:
updating for the NITS removed the /lgtm 

would appreciate another review...and if you have the time a look at this other pr as well https://github.com/kubernetes-sigs/kustomize/pull/5778 😅